src/main/java/com/fastcampus/projectboard/config/JpaConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.fastcampus.projectboard.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+import java.util.Optional;
+
+@EnableJpaAuditing
+@Configuration
+public class JpaConfig {
+
+    @Bean
+    public AuditorAware<String> auditorAware() {
+        return () -> Optional.of("authenticated user"); //TODO: Spring Security 인증 기능 추가되면 수정 필요
```

Comment:
여기서 이름은 실제로 사용자 데이터의 일부로 기록될 것이니까, 좀 더 간단하고 읽기 좋은 것으로 해도 돼요.

```suggestion
        return () -> Optional.of("코양이"); //TODO: Spring Security 인증 기능 추가되면 수정 필요
```

Reply:
@hegunhee 님 이 코드 참고해 보세요.

Reply:
주석에 Security가 달려있어서 추후에 작성되어도 될거라 생각했습니다.

해당 클래스 분석해보고 코드해 추가해놓겠습니다